geometry_msgs/Pose currentPose
---
sensor_msgs/PointCloud2 currentViewPointcloud
